dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/model/ResourceLink.java

```diff
@@ -339,4 +338,13 @@ public static boolean isDownloadRestricted(final String fileAssetName, final Con
         return false;
     }
 
+    /**
+     * Centralized code to get path
+     * @param contentlet
+     * @return
+     */
+    public static String getPath(Contentlet contentlet){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'contentlet' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5596402)